app/views/admin/governments/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<% page_title "Governments" %>
+<h1>Governments</h1>
+
+<% if can?(:create, Government) %>
+  <p>
+    <%= link_to "Create a government", new_admin_government_path %>
+  </p>
+<% end %>
+
+<table class="governments table table-striped table-bordered add-top-margin">
+  <thead>
+    <tr class="table-header">
+      <th width="70%">Name</th>
+      <th width="15%">Start date</th>
+      <th width="15%">End date</th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @governments.each do |government| %>
+      <%= content_tag_for(:tr, government) do %>
+        <td>
+          <% if can?(:edit, government) %>
+            <%= link_to government.name, edit_admin_government_path(government) %>
+          <% end %>
```

Comment:
Should there be an else which just lists the `government.name` otherwise you would just get a list of dates without titles?